docs/sources/clients/promtail/configuration.md

```diff
@@ -157,10 +157,9 @@ The `server` block configures Promtail's behavior as an HTTP server:
 [health_check_target: <bool> | default = true]
 ```
 
-## clients
+## client_config
 
-The `clients` block configures how Promtail connects to instances of
-Loki:
+`client_config` configures how Promtail connects to instances of Loki:
```

Comment:
I'm not sure this is correct.

The top level config keys appear to be `client` and `clients` (https://github.com/grafana/loki/blob/51fbc95bbd27b9edc584c66fd7d18256e11ec33a/clients/pkg/promtail/config/config.go#L30), so we shouldn't change that here.

Reply:
True, my suggestion is misleading. My intent was actually to say that the `<client_config>` type configures Promtail connection to Loki.